snappy_pipeline/apps/tpls/pipeline_job.sh

```diff
@@ -51,6 +51,7 @@ mkdir -p ${TMPDIR}
 test -z "${SLURM_JOB_ID-}" && SLURM_JOB_ID=$(date +%%Y-%%m-%%d_%%H-%%M)
 LOGDIR=slurm_log/${SLURM_JOB_ID}
 mkdir -p ${LOGDIR}
+export SBATCH_DEFAULTS=" --output=${LOGDIR}/%%x-%%j.log"
```

Comment:
Right now it would result in something like:
```
slurm_log
|-- 5125393
|   |-- snakejob.variant_export_varfish_annotator_annotate.101.sh-5125423.log
...
|   `-- snakejob.variant_export_varfish_annotator_annotate.92.sh-5125406.log
`-- variant_export-5125393.log
```

What about we prettify it by just keeping `slurm-<PIPELINE_NAME>-<JOB_ID>`? That would also keep it consistent with the original version. Untested:
```
export SBATCH_DEFAULTS=" --output=${LOGDIR}/slurm-%(step_name)s-%%j.log"
```

Reply:
I am not sure. I like the fact that the rule name & snakemake job id are embedded in the filename. It allows to to quickly find a job that has been restarted because it failed. I have for example:

```
$ ls -altr slurm_log/5033314
total 224
drwxr-sr-x 1 blance_c 1005272  4096 Nov  2 15:50 ..
-rw-rw---- 1 blance_c 1005272 25043 Nov  2 15:51 snakejob.ngs_mapping_target_coverage_report_collect.419.sh-5033316.log
-rw-rw---- 1 blance_c 1005272 21372 Nov  2 16:40 snakejob.ngs_mapping_star_run.59.sh-5033317.log
-rw-rw---- 1 blance_c 1005272 26447 Nov  2 16:41 snakejob.ngs_mapping_gatk_post_bam_run.58.sh-5034208.log
-rw-rw---- 1 blance_c 1005272 26447 Nov  2 16:42 snakejob.ngs_mapping_gatk_post_bam_run.58.sh-5034219.log
-rw-rw---- 1 blance_c 1005272 26447 Nov  2 16:43 snakejob.ngs_mapping_gatk_post_bam_run.58.sh-5034239.log
drwxrws--- 1 blance_c 1005272  4096 Nov  2 16:44 .
-rw-rw---- 1 blance_c 1005272 26447 Nov  2 16:44 snakejob.ngs_mapping_gatk_post_bam_run.58.sh-5034245.log
-rw-rw---- 1 blance_c 1005272 52258 Nov  2 19:22 snakejob.ngs_mapping_generate_doc_files.250.sh-5034207.log
```

But I see the appeal of consistency.